doc/dev/strava-api-calls.md

```diff
@@ -0,0 +1,44 @@
+# Strava API Calls Overview
+
+## Relevant Files
+
+- **Component**: `src/components/routes/StravaApp.vue`
+- **Store**: `src/stores/strava.ts`
+
+## API Calls
+
+### 1. Initial Component Mount (onMounted)
+
+- **API Call**: `GET /strava-account` (via `stravaStore.loadAccount()`)
+- **Purpose**: Load existing Strava account data if any
+- **Triggered**: When StravaApp component mounts and no account exists in store
+
+### 2. Connect New Account Flow
+
+- **Step 1**: `GET /strava-auth-url?scope={read|read_all}` (via `stravaStore.loadAuthUrl()`)
+  - Purpose: Get Strava OAuth authorization URL
+  - User clicks "Link to App" button
+- **Step 2**: Browser redirects to Strava for OAuth
+- **Step 3**: `GET /strava-auth?code={auth_code}` (via `stravaStore.authAccount()`)
+  - Purpose: Exchange OAuth code for account connection
```

Comment:
Getting User Strava account token (access, refresh) and creating User Strava account in our DB. And synchronize User Strava account data if some exists.

REST API URL endpoint return JSON with structure:

```Python
        response = {
            "account_status": "created" if created else "updated",
            "account": strava_account,
        }
```